cloud-aws/src/main/resources/templates/aws-cf-stack.ftl

```diff
@@ -25,8 +25,7 @@
       "Description" : "Id of the VPC where to deploy the cluster",
       "Type" : "String",
       "MinLength": "12",
-      "MaxLength": "12",
-      "AllowedPattern" : "vpc-[a-z0-9]{8}"
+      "AllowedPattern" : "vpc-[a-z0-9]*"
```

Comment:
maybe instead of `*` which is pretty permissive you could use:
`vpc-[a-z0-9]{8,}`
which defines only the minimum